packages/core/src/polar/index.ts

```
@@ -81,89 +28,69 @@ export namespace Polar {
-    //         teamID: input.teamID,
-    //         customerID: input.customerID,
-    //         subscriptionID: input.subscriptionID,
-    //         subscriptionItemID: input.subscriptionItemID,
-    //         standing: input.standing,
-    //     };
-    // }
+    //TODO: Implement this
+    export const handleWebhook = async(payload: ReturnType<typeof validateEvent>) => {
+        switch (payload.type) {
+            case "subscription.created":
+                const teamID = payload.data.metadata.teamID
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)